apps/web/src/hooks/__tests__/useInstrumentVisualization.test.ts

```
@@ -0,0 +1,140 @@
+      edition: 1,
+      name: 'test'
+    }
+  }))
+);
+
+const mockStore = {
+  currentGroup: { id: 'testGroupId' },
+  currentUser: { username: 'testUser' }
+};
+
+const mockBasicIsoString = '2025-04-30';
```

ESLint

[error] 'mockBasicIsoString' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)